core/src/main/java/de/danoeh/antennapod/core/sync/SyncService.java

```diff
@@ -69,29 +73,39 @@
     private static final String WORK_ID_SYNC = "SyncServiceWorkId";
     private static final ReentrantLock lock = new ReentrantLock();
 
-    private ISyncService syncServiceImpl;
+    private final ArrayList<ISyncService> syncServices = new ArrayList<>();
```

Comment:
I think it would be more robust to only allow one sync service at a time

Reply:
will do. I did not get so far to test when both services are enabled. But having only a single one active is easier to implement.

Reply:
done